packages/scaffold-config/src/frameworks.ts

```diff
@@ -61,6 +61,22 @@ function getBundlerDependency (bundler: WizardBundler['type'], projectPath: stri
   }
 }
 
+export const WIZARD_MOUNT_MODULES = ['cypress/react', 'cypress/react18', 'cypress/vue', 'cypress/vue2', 'cypress/angular'] as const
+
+export type WizardMountModule = typeof WIZARD_MOUNT_MODULES[number]
+
```

Comment:
I think you can remove `async` here, implied by the `Promise.resolve`

Reply:
Removed async in [143a9d1](https://github.com/cypress-io/cypress/pull/23251/commits/143a9d171d0389a749eac7907842e2d59390a6cf)